app/femr/data/DataModelMapper.java

```diff
@@ -363,6 +363,7 @@ public IPatient createPatient(int userID, String firstName, String lastName, Dat
         patient.setUserId(userID);
         patient.setFirstName(firstName);
         patient.setLastName(lastName);
+        patient.setPhoneNumber(phoneNumber);
```

Comment:
still getting it (the first 2 results are an empty string from not entering anything):

mysql> select p.phone_number, pe.date_of_triage_visit FROM patients p INNER JOIN patient_encounters pe ON pe.patient_id = p.id WHERE p.phone_number IS NOT NULL; 
+--------------+----------------------+
| phone_number | date_of_triage_visit |
+--------------+----------------------+
|              | 2016-11-30 17:41:00  |
|              | 2016-11-30 21:25:10  |
| 55522278989  | 2016-11-30 21:30:20  |
+--------------+----------------------+
3 rows in set (0.01 sec)